wordpress-mobile/WordPress-Android
# Feature/14745 dynamic card for blogging reminders

Fixes #14745 

TODO:
- Replace the image
- Style the button
- Update logic with feature flag check and seen already check

To test:

- Turn on the blogging reminders flag (Me/App settings/Test feature configuration - BloggingRemindersFeatureConfig)✅
- Turn on My Site Improvement flag (Me > App Settings > Test feature configuration > MySiteImprovementsFeatureConfig ✅

- Open My Site.  It should show under Site Information Block, a [Blogging Reminders Card](https://wpmobilep2.wordpress.com/2021/05/13/blogging-reminders-some-concepts-ideas-and-flows/)

<img width=320 src=https://user-images.githubusercontent.com/990349/120961443-3226d580-c7a1-11eb-91fc-134361c39b26.png>



## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
